executor/distsql.go

```diff
@@ -551,24 +514,28 @@ type IndexLookUpExecutor struct {
 	*dataReaderBuilder
 	// All fields above is immutable.
 
-	indexWorker
-	tableWorker
-	finished chan struct{}
-
-	// batchSize is for slow startup. It will slowly increase to the max batch size value.
-	batchSize int
+	idxWorkerWg sync.WaitGroup
+	tblWorkerWg sync.WaitGroup
+	finished    chan struct{}
 
 	resultCh   chan *lookupTableTask
 	resultCurr *lookupTableTask
 }
 
 // indexWorker is used by IndexLookUpExecutor to maintain index lookup background goroutines.
 type indexWorker struct {
-	wg sync.WaitGroup
+	workCh    chan<- *lookupTableTask
+	finished  <-chan struct{}
+	resultCh  chan<- *lookupTableTask
+	keepOrder bool
+
+	// batchSize is for slow startup. It will slowly increase to the max batch size value.
```

Comment:
1. batchSize's grow speed is not slow, It is doubled every time we call `extractTaskHandles`
2. "be increased to"